docker-compose.yml

```
@@ -0,0 +1,17 @@
+name: ultrasinger
+services:
+    ultrasinger:
+        deploy:
+            resources:
+                reservations:
+                    devices:
+                        - driver: nvidia
+                          count: all
+                          capabilities:
+                              - gpu
+        stdin_open: true
+        tty: true
+        container_name: UltraSinger 
+        volumes:
+            - ./src/output:/app/src/output
+        image: ultrasinger
```

use version from docker hub

```suggestion
        image: rakuri255/ultrasinger:latest
```

---

done it now uses the docker image